### DISCUSSION TITLE
Can't quote price

### DISCUSSION THREAD

**Discussion by ericmsott** [uniswap-python/uniswap-python#156] [Q&A] [answered]
I'm trying o use the function 'get_price_input', but it is raising the following error:

`    raise ValidationError(message)
web3.exceptions.ValidationError: 
Could not identify the intended function with name `quoteExactInputSingle`, positional argument(s) of type `(<class 'str'>, <class 'str'>, <class 'int'>, <class 'float'>, <class 'int'>)` and keyword argument(s) of type `{}`.
Found 1 function(s) with the name `quoteExactInputSingle`: ['quoteExactInputSingle(address,address,uint24,uint256,uint160)']
Function invocation failed due to no matching argument types.`

**Comment by sweeta20793** [comment, marked as answer]
Hey, try to use int(value) to resolve.
